File: pclassic/stubs-2016/DotMatrixChecker.java
import java.util.*;
import java.io.*;


public class DotMatrixChecker {
	//DONT EVER TOUCH THESE SHITS - THIS WILL BE PROVIDED DURING COMPEITTION
    public static void main(String args[]) throws FileNotFoundException, UnsupportedEncodingException
    {
        Scanner sc = new Scanner(new FileReader("DotMatrixCheckerIN.txt"));
        
        while (sc.hasNext()) {
            System.out.println(makeLogo(Integer.parseInt(sc.nextLine())));
        }
        
        
    }
    public static String makeLogo(int n)
    {
        String out = "";
        //write ur code here
               
        return out; 
    }
    //few extra notes - can use StringBuilder
}

File: pclassic/stubs-2016/DotMatrixChecker.py
def draw_dot_matrix_letter(N):
    letter = ""
    #write ur code here
    
    return letter

#DONT CHANGE THIS SHIT - THIS WILL BE PROVIDED (I DONT KNOW ENOUGH PYTHON TO FIX IT)
if __name__ == "__main__":
    with open("DotMatrixCheckerIN.txt", "r") as f:
        while True:
            s = f.readline()
            if s == '':
                break
            N = int(s)
            print(draw_dot_matrix_letter(N))
